src/api/python/cvc5.pxi

```diff
@@ -2377,21 +2377,26 @@ cdef class Solver:
             diffi[termk] = termv
         return diffi
 
-    def getValue(self, Term t):
+    def getValue(self, term_or_list):
         """
-            Get the value of the given term in the current model.
+            Get the value of the given term or list of terms in the current
+            model.
 
             SMT-LIB:
 
             .. code-block:: smtlib
 
-                ( get-value ( <term> ) )
+                ( get-value ( <term>* ) )
```

Comment:
Shouldn't this be `+`?

Reply:
This implementation is perfectly fine with an empty list.